script/security-check.js

```diff
@@ -69,6 +69,7 @@ function runAudit() {
   let auditOutput = '';
 
   child.stdout.setEncoding('utf8');
+  child.stderr.setEncoding('utf8');
```

Comment:
Ahh, yeah. That'll won't fix the issue from earlier today, but it'll format the error message better anyhow. Might still be worth adding a message in the `stderr.on` callback's `console.log`, but this gets us further, I think.